LMGeocoder/LMGeocoder.m

```diff
@@ -85,10 +99,13 @@ - (void)geocodeAddressString:(NSString *)addressString
                 NSString *urlString = kGoogleAPIGeocodingURL(addressString);
                 if (self.googleAPIKey != nil) {
                     urlString = kGoogleAPIURLWithKey(urlString, self.googleAPIKey)
+                    if (countryCode != nil) {
+                        urlString = kGoogleAPIURLAppendCountry(urlString, countryCode);
```

Comment:
Adding this comment to underline readability issue: 
Me as a reader, I assume this would be a method, written in swift. I'm confused with the `k` in front which does not look like a method, more like a constant or something and have to look it up. This all leads to a distraction while performing the code review or while generally inspecting and trying to understand the code :)